src/styles/main.scss

```
@@ -1,3 +1,4 @@
+@import "blocks/variables";
```

```suggestion
@import "utils/variables";
```
This is not a block, it better move to a more specific folder.